src/manifest.ts

```diff
@@ -0,0 +1,57 @@
+import auth0 from './lib/client'
+import { getAllClients, getValueAndComment } from './lib/utils'
+
+/**
+ * List of rules that should exist on the Auth0 tenant.
+ *
+ * Rules will be executed in the order they are defined
+ */
+const MANIFEST: RuleDefinition[] = [
+  {
+    name: 'Add email to access token',
+    file: 'email-to-access-token',
+    enabled: true,
+  },
+  {
+    name: 'Add Default Role To All Users',
+    file: 'add-default-roles',
+    enabled: true,
+  },
+  {
+    name: 'Filter scopes',
+    file: 'filter-scopes',
+    enabled: true,
+    getData: async () => {
```

Comment:
Wait, I have a dumb idea. Why do we need this to be dynamic? Can't we just use their ids and do a lookup based on tenant id?
```js
whitelistAll = {
  effective-altruism: ['abcd', 'efgh'],
  effective-altruism-local: ['ijkl', 'mnop'],
  ...
}
```
Then we could get rid of templating entirely.

Reply:
As discussed, we now use the `TEMPLATE_DATA` injection rather than Handlebars. This preserves the flexibility of being able to inject dynamic data without having to do any weird string coercion shenanigans. See the README for more detail.